File: SurgeonPortal.Models/CE/ExamScoreModel.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace SurgeonPortal.Models.CE
{
    public class ExamScoreModel
    {
        public int ExamScheduleScoreId { get; set; }
        public int ExamScheduleId { get; set; }
        public int ExaminerUserId { get; set; }
        public int ExaminerScore { get; set; }
        public bool? Submitted { get; set; }
        public DateTime? SubmittedDateTimeUTC { get; set; }
        public int CreatedByUserId { get; set; }
        public DateTime CreatedAtUtc { get; set; }
        public int LastUpdatedByUserId { get; set; }
        public DateTime LastUpdatedAtUtc { get; set; }
    }
}

File: SurgeonPortal.Api/ClientApp/src/app/api/models/ce/exam-score.model.ts

export interface IExamScoreModel {
    examScheduleScoreId: number;
    examScheduleId: number;
    examinerUserId: number;
    examinerScore: number;
    submitted: boolean;
    submittedDateTimeUTC: string;
    createdByUserId: number;
    createdAtUtc: string;
    lastUpdatedByUserId: number;
    lastUpdatedAtUtc: string;
}
